### ISSUE TITLE
navigation between collected material and special analysis module

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#1818]
Came out of PASU: The special ceramics module is too far away from the collected material. 
We should have a navigation button to go there and back.
- prerequisite: #1795

**Comment by urapadmin** [comment]
### recording 13.6.7
- If a ceramics module is configured and the material type of the cm equals that in "visibility"/"show_only_this_material_in_pottery_selection" a little vessel-button should appear next to the collected material.
- if also the ceramics preprocessing is on, the same button appears in the preprocessing pane, just larger. 
The two buttons do the exact same thing:
- jump to the configured ceramic module. Can be su_ceramics_processing or pottery_processing. In both cases the collected material is selected in the analysis module. If that collected material had never been analyzed before, a question like "do you really want to start analyzing this ..." makes sure that's intended.

**Comment by luizaogs** [comment]
This is working perfectly for urap ceramics; I haven't tried to test ustp yet because meh configuration.

**Comment by luizaogs** [comment]
Ok, for su ceramics: navigating back and forth is working fine. BUT I cannot figure out how to turn on the preprocessing thing (I have tried). So @lbestock can you just check that part?

**Comment by lbestock** [comment]
Now that one can navigate straight to ceramics processing from the CM, is it possible to also make it so one can create a ceramics module for that CM without going to registration? Do we want that? (I ask because the pot button to go to a ceramic record is there in the CM whether or not there is any pot record to go to.)

**Comment by lbestock** [comment]
We caught why it is so difficult to turn ceramics pre-processing on (I had to ask, too): there is a lurking plural in one of the configuration values. Now documented [here](https://docs.google.com/document/d/1xHvCOrYEdwYgamjCjdbh-L8Lfiv6uvot/edit).

**Comment by lbestock** [comment]
> Now that one can navigate straight to ceramics processing from the CM, is it possible to also make it so one can create a ceramics module for that CM without going to registration? Do we want that? (I ask because the pot button to go to a ceramic record is there in the CM whether or not there is any pot record to go to.)

OK this is weird. I made this note when I had things configured for URAP. When I configured them for PASU preprocessing and processing, I found that I COULD make such a record from within the CM. So clearly the answer to the above question is "yes" we want that and you anticipated that, but it wasn't working from PVD configured as URAP and I might now have to go test that separately and make it a new ticket, first I will get on top of the actual issue here.

**Comment by luizaogs** [comment]
I am pretty sure it worked for me on URAP ceramics, otherwise I would have made a note. I’ll check again.

**Comment by lbestock** [comment]
No need for you to test. It was a different problem. If the ceramic_module is misset to "su_ceramic_processing" instead of "su_ceramics_preprocessing" then neither is pre-processing visible nor is there a ceramics module at all. So either hitting the pot button in CM or trying to get to the ceramics module through registration results in an error message. Just a matter of configuration needing to be right, not a bug at all. 

Everything works as it should.
